components/camel-pulsar/src/main/java/org.apache.camel.component.pulsar/utils/retry/ExponentialRetryPolicy.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.pulsar.utils.retry;
+
+public class ExponentialRetryPolicy implements PulsarClientRetryPolicy {
+
+    @Override
+    public void retry() {
+        // TODO
```

Comment:
This doesn't do anything. Since it is called here https://github.com/apache/camel/pull/2849/files#diff-0b5e1073c757a871674be324659401b2R43

Probably it is better to use a different Default Retry Policy or remove this.

Reply:
removed